main.go

```diff
@@ -283,6 +282,8 @@ func main() {
 	archiveZipPath := filepath.Join(cfg.OutputDir, cfg.ArtifactName+".xcarchive.zip")
 	ideDistributionLogsZipPath := filepath.Join(cfg.OutputDir, "xcodebuild.xcdistributionlogs.zip")
 
+	shipZipPath := filepath.Join(os.Getenv("BITRISE_DEPLOY_DIR"), cfg.ArtifactName+".xcarchive.zip")
```

Comment:
"ship" has no meaning in this context, this should be archiveExportPath or similar